trompace/connection.py

```diff
@@ -72,3 +72,17 @@ async def download_file(url, file_link):
             for chunk in r.iter_content(chunk_size=8192):
                 if chunk:
                     f.write(chunk)
+
+def download_file(url, file_link):
```

Comment:
small thing, let's change the argument file_link to `filename` to better describe its purpose, here and in the async version